entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -602,6 +602,16 @@ class EntityService(
         return neo4jRepository.deleteEntity(entityId)
     }
 
+    fun deleteEntityAttribute(entityId: String, attributeName: String, contextLink: String): Pair<Int, Int> {
+        val expandedAttributeName = expandJsonLdKey(attributeName, contextLink)!!
+
+        if (neo4jRepository.hasPropertyOfName(entityId, expandedAttributeName))

```

Comment:
I think it could be straightforward if you just do a method getAttributeOfSubject (see getPropertyOfSubject) and work with Attribute type and then delete all linked properties with deleteAll(list)

What do you think ?

Reply:
I'm not sure, giving an entityId and an attribute name, we can have a generic method that returns the attribute instance.
Moreover, i don't see how we can delete this attribute and its attributes with the deleteAll() function.

Reply:
I don't know if that's possible I will check it out but I got the idea from the getPropertyOfSubject already written.